ToDoList-Redux/Binders/TaskListViewModel.swift

```diff
@@ -37,25 +30,24 @@ extension TaskList {
         case add(String)
         case remove(IndexSet)
         case move(IndexSet, Int)
-        case toggle(CheckmarkCellView.Action)
```

Comment:
Because the action "toggle" is handled completed by the cell, and because now I'm creating the cell from outside and injecting it here, we don't have to be the proxy for this action any more.